src/AeroSharp/DataAccess/ReadConfiguration.cs

```diff
@@ -78,5 +80,11 @@ public ReadConfiguration(ReadConfiguration other)
         /// If set to 1, batches are read sequentially. Must be >= 1.
         /// </summary>
         public int MaxConcurrentBatches { get; set; }
+        /// <summary>
+        /// Maximum number of concurrent synchronous batch node request threads to server nodes.
+        /// Asynchronous batch requests ignore this field and always issue all node requests in parallel.
+        /// Must be >= 0.
```

Comment:
It may be worth talking about the tradeoffs of setting this at 0 or 1 in our docs here. For example, according to the [Aerospike docs](https://docs.aerospike.com/apidocs/csharp/html/f_aerospike_client_batchpolicy_maxconcurrentthreads), setting this to 0 gives a performance advantage for large batch sizes but you also risk threadpool starvation:

> The downside is extra threads will need to be created (or taken from a thread pool). In extreme cases, the operating system's thread capacity could be exhausted.

Reply:
I don't have a strong opinion here, but if we adjust this at some point later my preference is to provide a blurb about what this does and encourage folks read the actual docs by linking off to them. Going verbatim well eventually bite us when things change.

Reply:
Sounds good y'all.  Will go for a future update to the language to check the docs for more insight.